proto/panacea/datapool/v2/pool.proto

```diff
@@ -9,17 +9,18 @@ import "google/protobuf/duration.proto";
 
 // Pool defines a data pool
 message Pool {
-  uint64                   pool_id = 1;
-  string                   pool_address = 2;
-  uint64                   round = 3;
-  PoolParams               pool_params = 4;
-  uint64                   cur_num_data = 5;                            // current number of data
-  uint64                   num_issued_nfts = 6;                         // current issued NFTs
-  string                   status = 7;
-  string                   curator = 8;
-  cosmos.base.v1beta1.Coin deposit = 9 [(gogoproto.nullable) = false]; // deposit
-  string                   nft_contract_addr = 10;                     // NFT contract address of pool
-  bool                     is_paid_deposit = 11;
+  uint64                                pool_id = 1;
+  string                                pool_address = 2;
+  uint64                                round = 3;
+  PoolParams                            pool_params = 4;
+  uint64                                cur_num_data = 5;                                        // current number of data
+  uint64                                num_issued_nfts = 6;                                     // current issued NFTs
+  string                                status = 7;
+  string                                curator = 8;
+  cosmos.base.v1beta1.Coin              deposit = 9 [(gogoproto.nullable) = false];             // deposit
+  string                                nft_contract_addr = 10;                                 // NFT contract address of pool
+  bool                                  is_paid_deposit = 11;
+  map<uint64, cosmos.base.v1beta1.Coin> curator_commission = 12 [(gogoproto.nullable) = false]; // key: round, value: paid curator commission balance
```

Comment:
This value stores the commission paid to the curator by round.
This value is used to calculate the commission distributed to the curator whenever the proceeds are distributed.